src/components/BrandConfigProvider/theme/Theme.ts

```diff
@@ -0,0 +1,24 @@
+import * as baseTheme from './base';
+
+interface Props {
+  colors?: Partial<baseTheme.Colors>;
+  spacing?: Partial<baseTheme.Spacing>;
+}
+
+export class Theme {
+  colors: baseTheme.Colors = baseTheme.colors;
+  spacing: baseTheme.Spacing = baseTheme.spacing;
```

Comment:
I think you might want to spread the baseTheme to avoid `mergeX` mutating the baseTheme values

Reply:
Thanks!